src/textual/_doc.py

```diff
@@ -45,9 +46,9 @@ def format_svg(source, language, css_class, options, md, attrs, **kwargs) -> str
 def take_svg_screenshot(
     app: App | None = None,
     app_path: str | None = None,
-    press: Iterable[str] = ("_",),
+    press: Iterable[str] = (),
     title: str | None = None,
-    terminal_size: tuple[int, int] = (24, 80),
+    terminal_size: tuple[int, int] = (80, 24),
```

Comment:
Feels surprising - usually, I see this as (rows, columns).

Reply:
It is directly translated to our Size object which is (width, height). Didn't want to have two formats for the same thing.